ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -281,11 +286,13 @@ const AssetListControlBar = ({
         : (currentMultichainNetwork.network.nickname ?? t('currentNetwork'));
     }
 
+    // NOTE - a little lazy here with this evalution
+    // We could have: enabledNetworks if selected > 1; allNetworks if equal to all active popular networks
```

Comment:
We can remove this, I think it is relatively safe to assume >1 networks should show 'all networks' text